### PR TITLE
now-using-django3

### PR DESCRIPTION
**Pull request by aisamuel**
Now using django 3.0 and python 3.8.5 versions

### CODE REVIEW

**Review by Alfaxad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ $ docker rm data-africa
 ```
 
 ### Production
+ 
+$ docker build -t registry.heroku.com/frozen-basin-50948/web .
 
-docker build -t registry.heroku.com/frozen-basin-50948/web .
+$ docker push registry.heroku.com/frozen-basin-50948/web
```
Comment (Damephena) [inline README.md:39]: Kindly rename the heroku app to something readable.

**Review by Damephena — COMMENTED** [review]
Heroku app rename

**Review by captbilard — APPROVED** [review]

**Review by captbilard — CHANGES_REQUESTED** [review]
Kindly change the heroku name as suggested

**Review by Damephena — CHANGES_REQUESTED** [review]
Just the heroku stuff then we're good.

**Pull request merged**